node_modules/gh-rest/lib/rest/user.js

```diff
@@ -82,6 +82,7 @@ module.exports = function(client) {
      * @param  {Boolean}        [opts.isAdmin]                  Whether the user is an app administrator
      * @param  {String}         [opts.recaptchaChallenge]       The identifier for the recaptcha challenge. Only required when the current user is not an app administrator
      * @param  {String}         [opts.recaptchaResponse]        The recaptcha response entered for the presented challenge. Only required when the current user is not an app administrator
+     * @param  {Boolean}        [opts.termsAndConditions]       Whether the user accepts the Terms and Conditions
```

Comment:
This should be `{Date}` I think. Maybe the language could say:

`When the user last accepted the terms and conditions, if at all`